smoketest/scripts/cli/test_service_dns_dynamic.py

```diff
@@ -26,6 +26,7 @@
 DDCLIENT_SYSTEMD_UNIT = '/run/systemd/system/ddclient.service.d/override.conf'
 DDCLIENT_CONF = '/run/ddclient/ddclient.conf'
 DDCLIENT_PNAME = 'ddclient'
+DDCLIENT_INTERVAL = '300'
```

Comment:
We try to not hardcode any values anywhere. There is a new "way" to retrieve CLI defaults in Python:

```python3
>>> from vyos.xml_ref import default_value
>>> base_path = ['service', 'dns', 'dynamic']
>>> cli_default = default_value(base_path + ['interval'])
>>> cli_default
'300'

```

Please change the code accordingly

Reply:
Nice!